internal/match/json.go

```diff
@@ -163,7 +163,13 @@ func JSONCheckOff(wantKey string, wantItems []interface{}, mapper func(gjson.Res
 // item calling `fn`. If `fn` returns an error, iterating stops and an error is returned.
 func JSONArrayEach(wantKey string, fn func(gjson.Result) error) JSON {
 	return func(body []byte) error {
-		res := gjson.GetBytes(body, wantKey)
+		var res gjson.Result
+		if wantKey == "" {
```

Comment:
@kegsay I had to made this change since the response itself was an array of objects & I didn't find any function to use over there

Reply:
Okay, that is a great change, thank you!